examples/dynamo_integration/scripts/download_agent_leaderboard_v2.py

```
@@ -0,0 +1,311 @@
+            raw_dir.mkdir(parents=True, exist_ok=True)
+
+            with open(raw_dir / "tools.json", "w") as f:
+                json.dump(tools, f, indent=2)
+            with open(raw_dir / "personas.json", "w") as f:
+                json.dump(personas, f, indent=2)
+            with open(raw_dir / "adaptive_tool_use.json", "w") as f:
+                json.dump([dict(s) for s in scenarios_ds], f, indent=2)
+
+            logger.info("Saved raw data to %s", raw_dir)
+
+        except Exception as e:
```

Ruff (0.14.8)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)